core/bones/json.py

```diff
@@ -0,0 +1,58 @@
+import ast
+import json
+from viur.core.bones.base import ReadFromClientError, ReadFromClientErrorSeverity
+from viur.core.bones.raw import RawBone
+
+
+class JsonBone(RawBone):
+    """
+    This bone saves its content as a JSON-string, but unpacks its content to a dict or list when used.
+    """
+
+    type = "raw.json"
+
+    def __init__(self, indexed=False, multiple=False, languages=None, *args, **kwargs):
+        assert not multiple
+        assert not languages
+        assert not indexed
+        super().__init__(*args, **kwargs)
+
+    def serialize(self, skel, name, parentIndexed):
+        if value := skel.accessedValues.get(name):
```

Comment:
This is too lazy, as @sveneberth mentioned in https://github.com/viur-framework/viur-core/pull/558#pullrequestreview-1242394347.

Reply:
```suggestion
        if name in skel.accessedValues:
            skel.dbEntity[name] = json.dumps(skel.accessedValues[name])
```
Maybe so ?